cmake/addExecutable.cmake

```
@@ -18,6 +18,7 @@ macro(alpaka_add_executable In_Name)
     add_executable(${In_Name} ${ARGN})
 
     if(alpaka_ACC_GPU_CUDA_ENABLE)
+       ENABLE_LANGUAGE(CUDA)
```

```suggestion
       enable_language(CUDA)
```

Everything else in our CMake setup is lowercase.